extensions/wikia/ThemeDesigner/ThemeDesignerHelper.class.php

```diff
@@ -25,37 +53,7 @@ public static function isValidColor( $sColor ) {
 		}
 
 		// Last chance: array is not proper hash so maybe it is predefined color name
-
-		$aColorArray = [
-			'aliceblue', 'antiquewhite', 'aqua', 'aquamarine', 'azure',
-			'beige', 'bisque', 'black', 'blanchedalmond', 'blue', 'blueviolet', 'brown',
-			'burlywood', 'cadetblue', 'chartreuse', 'chocolate', 'coral', 'cornflowerblue',
-			'cornsilk', 'crimson', 'cyan', 'darkblue', 'darkcyan', 'darkgoldenrod',
-			'darkgray', 'darkgreen', 'darkkhaki', 'darkmagenta', 'darkolivegreen',
-			'darkorange', 'darkorchid', 'darkred', 'darksalmon', 'darkseagreen',
-			'darkslateblue', 'darkslategray', 'darkturquoise', 'darkviolet',
-			'deeppink', 'deepskyblue', 'dimgray', 'dodgerblue', 'firebrick',
-			'floralwhite', 'forestgreen', 'fuchsia', 'gainsboro', 'ghostwhite',
-			'gold', 'goldenrod', 'gray', 'green', 'greenyellow', 'honeydew',
-			'hotpink', 'indianred', 'indigo', 'ivory', 'khaki', 'lavender', 'lavenderblush',
-			'lawngreen', 'lemonchiffon', 'lightblue', 'lightcoral', 'lightcyan',
-			'lightgoldenrodyellow', 'lightgray', 'lightgreen', 'lightpink',
-			'lightsalmon', 'lightseagreen', 'lightskyblue', 'lightslategray',
-			'lightsteelblue', 'lightyellow', 'lime', 'limegreen', 'linen', 'magenta',
-			'maroon', 'mediumaquamarine', 'mediumblue', 'mediumorchid', 'mediumpurple',
-			'mediumseagreen', 'mediumslateblue', 'mediumspringgreen', 'mediumturquoise',
-			'mediumvioletred', 'midnightblue', 'mintcream', 'mistyrose', 'moccasin',
-			'navajowhite', 'navy', 'oldlace', 'olive', 'olivedrab', 'orange', 'orangered',
-			'orchid', 'palegoldenrod', 'palegreen', 'paleturquoise', 'palevioletred',
-			'papayawhip', 'peachpuff', 'peru', 'pink', 'plum', 'powderblue',
-			'purple', 'red', 'rosybrown', 'royalblue', 'saddlebrown', 'salmon',
-			'sandybrown', 'seagreen', 'seashell', 'sienna', 'silver', 'skyblue',
-			'slateblue', 'slategray', 'snow', 'springgreen', 'steelblue', 'tan',
-			'teal', 'thistle', 'tomato', 'turquoise', 'violet', 'wheat', 'white',
-			'whitesmoke', 'yellow', 'yellowgreen'
-		];
-
-		$isColorName = in_array( strtolower( $sColor ), $aColorArray );
+		$isColorName = in_array( strtolower( $sColor ), static::COLORS );
 
 		Wikia::log( __METHOD__, "JKU", "[{$sColor}] is not a valid color name" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace or remove this deprecated Wikia::log logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)